### ISSUE TITLE
Creating Kubeconfig file causes error

### ISSUE THREAD

**Issue by darkn3rd** [weaveworks/eksctl#2548]
**What happened?**

When creating a cluster with the `--kubeconfig` option, a kubeconfig file is not created, instead of error is produced (below).  This is due to an unpopulated kubeconfig file.  Currently, `eksctl`, a cluster provisioning tool, depends on others kubernetes clusters being provisioned outside of `eksctl`.  You shouldn't need clusters created before creating a cluster.

```
[✖]  parsing kubectl version string  (upstream error: error: Missing or incomplete configuration info.  Please point to an existing, complete config file:

  1. Via the command-line flag --kubeconfig
  2. Via the KUBECONFIG environment variable
  3. In your home directory as ~/.kube/config

To view or setup config directly use the 'config' command.
) / "0.0.0": Version string empty
```

**What you expected to happen?**

I expected a KUBECONFIG file to be created where specified.  Because I am provisioning a Kubernetes cluster for the first time with `eksctl`, there should not be a dependency on the existing cluster provisioned already (populated kubeconfig).  `eksctl` should use `kubectl version --client` if the version of the client tool is important.

**How to reproduce it?**

```bash
eksctl create cluster \
  --config-file "cluster.yaml" \
  --kubeconfig="demo-cluster-config.yaml"
```

```yaml
apiVersion: eksctl.io/v1alpha5
kind: ClusterConfig
metadata:
  name: my-ingress-demo-cluster
  region: us-west-2
  version: "1.14"
nodeGroups:
  - name: my-ingress-demo-cluster-workers
    instanceType: t3.medium
    desiredCapacity: 4
    minSize: 1
    maxSize: 4
    iam:
      withAddonPolicies:
        externalDNS: true
        certManager: true
```

```yaml
# ~/.kube/config
apiVersion: v1
clusters: []
contexts: []
current-context: ""
kind: Config
preferences: {}
users: []
```

**Anything else we need to know?**

Host OS: Ubuntu 19.10
Downloaded Release Tarball: 0.24.0

**Versions**
Please paste in the output of these commands:
```
$ eksctl version
0.24.0
$ kubectl version --client
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-16T11:56:40Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by michaelbeaumont** [comment]
@darkn3rd Can you try with a newer version of `kubectl` (at least `v1.18.3`)? It looks like [this issue was made](https://github.com/kubernetes/kubectl/issues/853#issuecomment-634869744) and was [fixed in `v1.18.3`](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.18.md#bug-or-regression-5)

**Comment by michaelbeaumont** [comment]
@darkn3rd Closing now, let us know if the issue remains after upgrading.
